filebeat/input/log/log.go

```diff
@@ -135,10 +139,10 @@ func (f *Log) errorChecks(err error) error {
 
 	if f.config.CloseRemoved {
 		// Check if the file name exists. See https://github.com/elastic/filebeat/issues/93
-		_, statErr := os.Stat(f.fs.Name())
+		stat, statErr := os.Stat(f.fs.Name())
 
-		// Error means file does not exist.
-		if statErr != nil {
+		// Error means file does not exist, or if inode has changed, then file was removed.
```

Comment:
If the inode of a file path has changed, it does not mean the file has been removed. It can also mean the file was removed. If you want to close a file this file you should use `close_renamed` (see check above).

Reply:
I tested set close_renamed to true, but the result dosen't what I want： after close the deleted  files twice，with the same name. Filebeat just stop handle the new file with the same name.(It means I can't find any '/tmp/test.log' in '/proc/filebeat_pid/fd')